client/src/actions/setSelectedMovie.js

```diff
@@ -0,0 +1,6 @@
+import {SET_SELECTED_MOVIE} from './actionTypes';
+
+export const setSelectedMovie = (movie) => ({
+    type: SET_SELECTED_MOVIE,
+    movie
```

Comment:
_trivial_
Accordingly to the Flux Standard action convention, when there is some additional data in action occur, it should be enclosed under `payload` property, e.g.

```javascript
{
  type: SET_SELECTED_MOVIE,
  payload: {
    movie,
  },
}
```

But after all, it's up to you, adhere to it or not.